Solutions/Alcide kAudit/SolutionMetadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"publisherId": "azuresentinel",
+	"offerId": "azure-sentinel-solution-isvtesting4",
+	"firstPublishDate": "2022-05-02",
+	"providers": ["Alcide"],
+	"categories": {
+		"domains" : ["Security - Threat Protection"]
+	},
+	"support": {
+	  "name": "Alcide",
+	  "email": "support@microsoft.com",
```

Comment:
@vmanojreddy - Please change email address it shouldn't be Microsoft.com

Reply:
Done